Formula/f/forgit.rb

```diff
@@ -13,17 +13,32 @@ class Forgit < Formula
 
   def install
     bin.install "bin/git-forgit"
-    bash_completion.install "completions/git-forgit.bash"
+    bash_completion.install "completions/git-forgit.bash" => "git-forgit"
+    zsh_completion.install "completions/git-forgit.zsh" => "_git-forgit"
     inreplace "forgit.plugin.zsh", 'FORGIT="$INSTALL_DIR', "FORGIT=\"#{opt_prefix}"
+    inreplace "conf.d/forgit.plugin.fish",
+              'set -x FORGIT "$FORGIT_INSTALL_DIR/bin/git-forgit"',
+              "set -x FORGIT \"#{opt_prefix}/bin/git-forgit\""
+    pkgshare.install "conf.d/forgit.plugin.fish"
     pkgshare.install "forgit.plugin.zsh"
     pkgshare.install_symlink "forgit.plugin.zsh" => "forgit.plugin.sh"
   end
 
   def caveats
     <<~EOS
-      A shell plugin has been installed to:
-        #{opt_pkgshare}/forgit.plugin.zsh
-        #{opt_pkgshare}/forgit.plugin.sh
+      A shell plugin has been installed. Please follow these instructions, then restart your shell.
```

Comment:
I much prefer the previous instructions, it is expected that people know what to do with a plugin

Reply:
Before I change this back (which I'm happy to do if you insist), I got this pattern from autojump:

https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/a/autojump.rb#L34-L40

I've run into a problem before when I'm installing something with `brew` that I expect to just do `brew install foo` and instantly be able to use it. I find it useful to have any additional instructions be explicit in the formula, so I can instantly see everything I need to do to get it to work.


Another example:
https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/t/thefuck.rb#L65-L69

Both `autojump` and `thefuck` I have personally messed up before, so these caveats were lifesavers. 



I'd like to state again, more than happy to revert if you insist, just stating my case for keeping it. We can document it on our side if this is unacceptable.


Other examples that are around (basically `grep "source.*HOMEBREW_PREFIX"`
- [zsh-autosuggestions.rb Lines 19-27](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-autosuggestions.rb#L19-L27)
- [zsh-history-substring-search.rb Lines 26-34](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-history-substring-search.rb#L26-L34)
- [zsh-autocomplete.rb Lines 20-29](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-autocomplete.rb#L20-L29)
- [zsh-syntax-highlighting.rb Lines 26-34](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-syntax-highlighting.rb#L26-L34)
- [zsh-autopair.rb Lines 18-26](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-autopair.rb#L18-L26)
- [zsh-navigation-tools.rb Lines 31-38](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-navigation-tools.rb#L31-L38)
- [zsh-you-should-use.rb Lines 16-24](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/z/zsh-you-should-use.rb#L16-L24)
- [powerlevel10k.rb Lines 26-34](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/p/powerlevel10k.rb#L26-L34)
- [root.rb Lines 169-178](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/r/root.rb#L169-L178)
- [antigen.rb Lines 19-30](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/a/antigen.rb#L19-L30)
- [autojump.rb Lines 33-43](https://github.com/Homebrew/homebrew-core/blob/9713251d101b91cf9f96d3a5e2c3331b165b492b/Formula/a/autojump.rb#L33-L43)

Reply:
The instructions on where to find files are fine, but having an entire manual is not. Caveats should only represent the specific differences between Homebrew and upstream instructions.

Reply:
Understood! Thanks for the feedback, reverted to the original version (plus fish) in this latest crack. 

Let me know if you'd like to see any other changes.